.eslintrc.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
             { "SwitchCase": 1 }
         ],
         "linebreak-style": [ "error", "unix" ],
-        "quotes": [ "error", "single" ],
+        "quotes": [ "error", "single", { "avoidEscape": true } ],
```

Comment:
This makes it possible to use double quotes when the string contains a single quote.